helm/attestation-verifier/charts/aas/templates/backup-volumes.yaml

```diff
@@ -11,6 +11,7 @@ metadata:
     "helm.sh/hook": pre-upgrade
     "helm.sh/hook-weight": "-4"
 spec:
+  ttlSecondsAfterFinished: 300

```

Comment:
are we expecting it takes 300 second to cleanup ? from where we got this number ?

Reply:
ttlSecondsAfterFinished defines the time to live once Job are finished and  I believe we can keep as low as 1 sec.
There would be some cron failed jobs and started automatically. 300 will allow to stay the job for a while and help in debug.